### ISSUE TITLE
Image chat not working

### ISSUE THREAD

**Issue by rkindustriesYT** [yatendra2001/ai_buddy#6]
Its just says waiting for response.

**Comment by yatendra2001** [comment]
yeah, that's a known issue. not getting enough time to fix. do you wanna go ahead and try a fix? i can help where ever you need?

**Comment by rkindustriesYT** [comment]
I am not a good developer but I would love to do that. I'll give it a try.

**Comment by yatendra2001** [comment]
yeah sure, please let me know any help you need from my side :)

**Comment by voidAnik** [comment]
Is anyone trying to fix this? Or can I?

**Comment by rkindustriesYT** [comment]
> Is anyone trying to fix this? Or can I?

You should
